### ISSUE TITLE
"OCI runtime create failed: invalid mount" after docker update

### ISSUE THREAD

**Issue by erikderzweite** [strukturag/docker-webrtc-turnserver#10]
I use turnserver with docker-compose. After an update to docker turnserver stopped working. The error message is:
`ERROR: for turnserver  Cannot start service turnserver: failed to create shim: OCI runtime create failed: invalid mount {Destination:= Type:bind Source:/var/lib/docker/volumes/916ff77ef43e036578eeda80bf1d985792d9795d1d0abcedadd3717683845001/_data Options:[rbind]}: mount destination = not absolute: unknown`

Here is the snippet of the docker-compose.yml:
```
  turnserver:
    restart: always
    image: spreed/turnserver:v0.3
    container_name: turnserver
    network_mode: "host"
    labels:
      - traefik.enable=false
    volumes:
      - /docker/turnserver:/srv
```

docker-version:
```
Client:
 Version:           20.10.2
 API version:       1.41
 Go version:        go1.13.8
 Git commit:        20.10.2-0ubuntu1~20.04.2
 Built:             Tue Mar 30 21:24:57 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.2
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.8
  Git commit:       20.10.2-0ubuntu1~20.04.2
  Built:            Mon Mar 29 19:10:09 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.2-0ubuntu1~20.04.1
  GitCommit:        
 runc:
  Version:          1.0.0~rc95-0ubuntu1~20.04.1
  GitCommit:        
 docker-init:
  Version:          0.19.0
  GitCommit:        
```

**Comment by RafNie** [comment]
The problem is related with the Docker file. I had the same problem on ubuntu after the update. I prepared a fix for this https://github.com/strukturag/docker-webrtc-turnserver/pull/11

**Comment by fancycode** [comment]
Fixed with #11
